Rolique/UI/Colleagues/ColleaguesTableViewCell.swift

```diff
@@ -51,82 +57,94 @@ class ColleaguesTableViewCell: UITableViewCell {
     todayStatusLabel.layer.cornerRadius = 4
     todayStatusLabel.setContentHuggingPriority(UILayoutPriority(251), for: .horizontal)
     
-    phoneImage.translatesAutoresizingMaskIntoConstraints = false
+    phoneImageView.translatesAutoresizingMaskIntoConstraints = false
   }
   
   required init?(coder aDecoder: NSCoder) {
     super.init(coder: aDecoder)
   }
   
-  func configure(with name: String, todayStatus: String?, title: String, isButtonEnabled: Bool, isMe: Bool) {
+  func configure(with user: User) {
```

Comment:
Your cell titly coupled to `User`. Now, you won't be able to reuse it for another type.